browser/src/app/MultiPageViewLayout.ts

```diff
@@ -0,0 +1,179 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+class LayoutPageRectangle extends cool.SimpleRectangle {
```

Comment:
Could you please add a one-liner documentation on what this class does? This is hopefully not much work and it saves a lot of time for the next person who has to understand this code.